src/pages/forgotpassword/ForgotPassword.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from "react-router-dom";
+
+import '../../common.css';
+
+function ForgotPassword() {
+  return (
+    <div className="content-container">
```

Comment:
What kind of content does this container wrap?
It seems like a very generic name and it is a great example that shows that you need to revisit your naming conventions. Maybe `forget-password-container`?

Reply:
this can maybe be removed. I don't know if there is a unified name for this. maybe page content instead? it is used in alle the signup/login/forgotpassword pages